AudioSensei/AvaloniaExtensions.cs

```diff
@@ -9,7 +9,7 @@ namespace AudioSensei
 {
     public static class AvaloniaExtensions
     {
-        public static void SetForegroundWindow(this Window window)
+        public static void SetForegroundWindow([JetBrains.Annotations.CanBeNull] this Window window)
```

Comment:
Why is there a namespace?

Reply:
I have literally no idea why.
It should obviously be:
```suggestion
        public static void SetForegroundWindow([CanBeNull] this Window window)
```